AUTHORS

```diff
@@ -12,4 +12,4 @@ Max Ekman <max@looplab.se>
 Rifiniti, Inc
 Stephan Sperber
 Thorsten von Eicken <tve@voneicken.com>
-
+Aaron Fischer <mail@aaron-fischer.net>
```

Comment:
Please put at the top to keep sorted.

Reply:
I tried the codespace thing to edit your PR but the flow failed. Obviously I can't push to your git repo, then it prompted to fork and failed because I already have a personal fork. Oh well.

Reply:
:) I've updated the AUTHORS and CONTRIBUTORS file accordingly. I did not get the difference between these two files tho. People who contribute code /are/ contributors and authors and hold copyright. Do I need to sign some paperwork?

Reply:
In theory yes, https://cla.developers.google.com/about/google-individual

Reply:
Okay, I submitted the CLA.